resources/js/Pages/Notes/Index.vue

```diff
@@ -0,0 +1,61 @@
+<script setup>
+import { defineProps, ref } from 'vue'
+import { Inertia } from '@inertiajs/inertia'
+import { TrashIcon  } from '@heroicons/vue/24/solid'
+const props = defineProps({
+  notes: Array,
+})
+
+const noteText = ref('')
+
+const saveNotes = () => {
+  const NewNote = props.notes[props.notes.length - 1]
+  const noteText = NewNote ? [NewNote.text] : []
+
+  Inertia.post('/notes', { noteText })
+}
+
+const addNote = () => {
+  if (noteText.value) {
+    // eslint-disable-next-line vue/no-mutating-props
+    props.notes.push({ text: noteText.value })
+    noteText.value = ''
+    saveNotes()
+  }
+}
+
+</script>
+
+
+<template>
+  <div>
+    <div class="note-container mb-4 flex">
```

Comment:
When we make some frontend stuff, we attach some screenshots, because most of people who makes code review doesn't run the application, they just read the code.

Reply:
![Screenshot_20230720_114410](https://github.com/blumilksoftware/escooters/assets/115737114/090110b9-0e3d-4600-80f0-b6023d482571)

Reply:
Please add the screenshot to pr description. Also please remember to write "this should close ..." where in place of ... the task number must be entered. After merging this pr to main, the issue will be automatically closed. 
Speaking of frontend, please look at admin panel - I think we should style the notes something like we have in admin panel. I'd rather not do another task for style this site.